ecosystem/opUSDC/op-usdc.md

```diff
@@ -0,0 +1,112 @@
+2. The `L2OpUSDCBridgeAdapter` burns the token.
+3. The `L2OpUSDCBridgeAdapter` sends the message to the appointed `CrossDomainMessenger`.
+4. The message is eventually included and proven on L1.
+5. Wait for the challenge period (at least 7 days).
+6. The receiving user (or relayers) withdraws the message after the challenge period, which is then forwarded to the `L1OpUSDCBridgeAdapter` that releases the specified amount of USDC to the user.
+
+### 3) Migrating to Native USDC
+
+Deprecation of the current system and upgrade of `BridgedUSDC` to native USDC involves the following steps:
+
+1. Circle communicates to the chain operator their intention to migrate to native USDC.
+2. The chain operator initiates `migrationToNative` to set the address for ownership transfer during the upgrade process. This will result in:
+    1. Setting the `circle` address in `L1OpUSDCBridgeAdapter`.
+    2. Transferring ownership of `BridgedUSDC` to Circle.
+    3. Stopping new messages from being sent on both adapters.
+    4. Triggering the `setBurnAmount` function to send back to L1. 
+3. Once settled, typically after at least seven days, Circle can call `burn` in `L1OpUSDCBridgeAdapter` to effectively burn the USDC supply in L1.
+4. Meanwhile, Circle can upgrade `BridgedUSDC` to native USDC.
+
+By doing so, Circle is fully onboarded in the OP Chain, and the old system is deprecated.
+
+### **Additional aspects**
+
+- Anyone can deploy an `L1OpUSDCFactory`, but only one is needed since it is permissionless and immutable. Additionally, it will return the deployment addresses when the `deploy` function is called.
+- `L1OpUSDCBridgeAdapter` and `L2OpUSDCBridgeAdapter` are not upgradable.
+- Once the migration process starts, it is irreversible.
+- In case of a failed transaction on L2 during migration, the`migrateToNative` function will remain open, allowing the executor to retry if needed.
+- Liability remains solely on each chain operator during the implementation until the migration process is completed.
+
+# Challenges ahead
+
+1. **Interop-compatibility**: A solution is still needed for cross-chain transfers between `BridgedUSDC` and other USDC versions implemented on other OP Chains, such as native ones. An intent-based design promises to be a viable solution for moving USDC across the Superchain.
+2. **Token pathfinding:** OP Chains must collaborate with bridge UI maintainers, third-party operators, and token list maintainers to provide accurate token bridging paths. This is important when transferring from one OP Chain to another where the USDC used is not fungible between them. Users would utilize a combination of CCTP and `OpUSDCBridgeAdapter`, passing through L1 to reach the destination and receive the exact amount bridged. However, L2-L2 transfers can be economically more convenient through a combination of both CCTP and Interop via intents. A well-designed token pathfinding system should ultimately help users save costs and avoid potential mistakes.
```

Comment:
To maintain the same contract addresses across the Superchain, I suggest all contracts are deployed through the [deterministic deployment proxy](https://github.com/Arachnid/deterministic-deployment-proxy).